content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2174,20 +2179,20 @@ These arguments are added to the relevant field names:
 
 #### Signature
 
-```prisma no-lines
-@@unique(_ fields: FieldReference[], name: String?, map: String?)
-```
+> ```prisma no-lines
+> @@unique(_ fields: FieldReference[], name: String?, map: String?)
+> ```
 
-> **Note**: Until version 3.0.0, the signature was:
+> **Note**: Before version 4.0.0, or before version 3.5.0 with the `extendedIndexes` Preview feature enabled, the signature was:
 >
 > ```prisma no-lines
-> @@unique(_ fields: FieldReference[], name: String?)
+> @@unique(_ fields: FieldReference[], name: String?, map: String?)
 > ```
 
-> **Note**: In version 3.5.0 and later, the signature for the `"extendedIndexes"` Preview feature is:
+> **Note**: Before version 3.0.0, the signature was:
 >
 > ```prisma no-lines
-> @@unique(_ fields: FieldReference[], name: String?, map: String?)
+> @@unique(_ fields: FieldReference[], name: String?)
 > ```
```

Comment:
Is the end result correct here? Seems a bit like it is all in a quote now.

Reply:
Here is the preview: 
<img width="557" alt="image" src="https://user-images.githubusercontent.com/183673/176185476-2c678a77-67e0-49c3-a49e-69031952f1ea.png">